traffic_ops/traffic_ops_golang/deliveryservice/deliveryservices_test.go

```diff
@@ -70,6 +70,9 @@ func TestGetDetails(t *testing.T) {
 	if oldDetails.OldCdnName != "foo" {
 		t.Errorf("expected old cdn name to be foo, but got %v", oldDetails.OldCdnName)
 	}
+	if *oldDetails.OldCdnId != 1 {
```

Comment:
You need to add a `nil` check for `oldDetails.OldCdnId` here. Basically something like is being done in line 61.